src/webgpu/shader/validation/wgsl/duplicate-var-in-nested-scopes.pass.wgsl

```diff
@@ -1,4 +1,4 @@
-// v-0014 - This fails because variable `a` is redeclared.
+// v-0014 - This passes because inter-scope shadowing is supported
```

Comment:
Nit: Please remove the "v-0014" part (WGLS no longer has validation error codes).

Reply:
done